Create ui-c8y-1020-32-0-changed-dm-title-to-be-title-cased.md (#2510)
SoftwareAG/c8y-docs: content/change-logs/device-management/ui-c8y-1020-32-0-changed-dm-title-to-be-title-cased.md

---
date: ""

> title: Changed DM title to be title cased.

Author: The Claude AI suggested to use the following title:
```suggestion
title: Device management titles now use title case
```

> title: Device management titles now use title case

Reviewer: ```suggestion
title: Device management title now uses title case as all the application titles.
```

product_area: Device management & connectivity
change_type:
  - value: change-QHu1GdukP
    label: Feature
component:
  - value: component--KIsStyzM
    label: Device Management app
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: DM-3890
version: 1020.32.0
---

> Changed DM title to be title cased.

Author: The Claude AI suggested to use the following body:
```suggestion
To improve readability and consistency, the titles of device management entities now use title case instead of sentence case. This change applies to all existing and new device management entities. The functionality of the entities remains the same, only the casing of the titles has been modified.
```

> To improve readability and consistency, the titles of device management entities now use title case instead of sentence case. The functionality of the entities remains the same, only the casing of the titles has been modified.

Reviewer: ```suggestion
To improve readability and consistency, the titles of device management entities now use title case (Device Management) instead of sentence case (Device management). The functionality of the entities remains the same, only the casing of the titles has been modified.
```

> To improve readability and consistency, the titles of device management entities now use title case (Device Management) instead of sentence case (Device management). The functionality of the entities remains the same, only the casing of the titles has been modified.

Reviewer: ```suggestion
To improve readability and consistency, the titles of device management application now use title case (Device Management) instead of sentence case (Device management). The functionality of the entities remains the same, only the casing of the titles has been modified.
```

> To improve readability and consistency, the titles of device management entities now use title case instead of sentence case. This change applies to all existing and new device management entities. The functionality of the entities remains the same, only the casing of the titles has been modified.

Reviewer 2: ```suggestion
To improve readability and consistency, the titles of device management entities now use title case (Device Management) instead of sentence case (Device management). This change applies to all existing and new device management entities. The functionality of the entities remains the same, only the casing of the titles has been modified.
```

> To improve readability and consistency, the titles of device management entities now use title case instead of sentence case. This change applies to all existing and new device management entities. The functionality of the entities remains the same, only the casing of the titles has been modified.

Reviewer: ```suggestion
To improve readability and consistency, the titles of device management entities now use title case instead of sentence case. The functionality of the entities remains the same, only the casing of the titles has been modified.
```
